components/server/src/main/antlr4/com/hotels/styx/server/routing/Condition.g4

```diff
@@ -50,6 +50,7 @@ DQUOTE : ["] ;
 ID : [a-zA-Z0-9_]+ ;
 
 // TODO: should also accept any visible non-delimiting US-ASCII character
+// Not clear what this comment is referring to. Need more information.
```

Comment:
Let's stick to 1.0 API related TODO items. This is not one of them. Please back off this change.